tasks/Cargo.toml

```diff
@@ -7,6 +7,10 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 imap-flow = { path = ".." }
 imap-types = "2.0.0"
+static_assertions = "1.1.0"
 tag-generator = { path = "../tag-generator" }
 thiserror = "1.0.58"
+tokio = { version = "1.37.0", features = ["net"] }
```

Comment:
Do we need tokio as non-dev dependency? I thought `Scheduler` is sans I/O now.

Reply:
No we don't, it's just dead code from refactors. I will remove it, nice catch!